Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM hmcts/cnp-java-base:openjdk-jre-8-alpine-1.3
```

Comment:
1.4 is out, where'd u get 1.3 from?

Reply:
got it from rhubarb recipe service :) will update